app/components/services/services.html

```diff
@@ -84,6 +90,10 @@
               <tr dir-paginate="service in (state.filteredServices = ( services | filter:state.filter | orderBy:sortType:sortReverse | itemsPerPage: state.pagination_count))">
                 <td><input type="checkbox" ng-model="service.Checked" ng-change="selectItem(service)"/></td>
                 <td><a ui-sref="service({id: service.Id})">{{ service.Name }}</a></td>
+                <td>
+                  <span ng-if="service.Status === 'partially running'" class="label label-{{ service.Status|servicestatusbadge }}" uib-tooltip="Some of replicas are not (yet) running">{{ service.Status }}</span>
```

Comment:
Is it possible to use [portinerTooltip](https://github.com/portainer/portainer/blob/develop/app/directives/tooltip.js) here instead? This will reduce the amount of code to refactor if we want to change something across all tooltips or move to another tooltip.